### PR TITLE
#163426611 Enable travis CI and add travis badge

### PR DESCRIPTION
**Pull request by Ndifreke**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - postgresql
+
+install:
+  - npm install
+
+script:
+  # - npm run test
+
+notifications:
+  slack: andela:E6PvFgt7ygC8QJHxXQJ7mEML
```
Comment (benfluleck) [inline .travis.yml]: Please create an environmental variable for this. This variable should not be exposed

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - psql -c 'create database travis_ci_test;' -U postgres
+
+after_script:
+ # -run after all script is executed successfully
+
+ after_success:
+  # - run all jobs is successfull
+
+after_failure:
+  # - run if any job fails
```
Comment (benfluleck) [inline .travis.yml]: A new line needed here. You can add a .editorconfig file to enforce this in all your files

File: .travis.yml (hunk 1 above)
Comment (benfluleck) [reply .travis.yml]: maybe a SLACK_ENV

**Review by benfluleck — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (Ndifreke) [reply .travis.yml]: @benfluleck this has been fixed

File: .travis.yml (hunk 1 above)
Comment (Ndifreke) [reply .travis.yml]: @benfluleck I have encrypted this string

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,3 @@
+end_of_line (on save)
+insert_final_newline (on save)
+trim_trailing_whitespace (on save)
```
Comment (benfluleck) [inline .editorconfig]: Please put a new line in the `editor.config`

**Comment by benfluleck** [comment]
@Ndifreke There are too many commits on this branch please squash them into 1 or 2 commits

**Pull request merged**